rfcs/text/0000-dss-metrics.md

```diff
@@ -0,0 +1,78 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC: Key performance metrics of the DCP Data Store (DSS)
+
+## Summary
+
+This RFC proposes a set of key performance metrics to be introduced and tracked as a summary measure of performance by
+which the utility of the Data Store can be evaluated, and by which informed data-driven improvement and/or regression
+decisions may be made.
+
+## Author(s)
+
+* [Andrey Kislyuk](mailto:akislyuk@chanzuckerberg.com)
+
+## Shepherd
+
+* [Andrey Kislyuk](mailto:akislyuk@chanzuckerberg.com)
+
+## Motivation
+
+Today, the DSS only tracks latency and error rates, which alone, does not provide a full picture of its utility.
+quantify higher level performance metrics that reflect the Data Store's ability to make HCA data accessible, computable,
+and durable.
+
+This RFC introduces a first iteration of DSS-specific key performance indicators (KPIs) that can be used to make DSS
+performance more measurable, and changes more observable and actionable. The suggested set of metrics is intended to be
+mutable; the service team should be empowered to adjust and expand the set as long as observability is not negatively
```

Comment:
Instead of stating "be empowered" which almost has a feel of giving permission, what about "the service team should adjust and expand the set over time while ensuring observability is not negatively impacted."?